Include/code.h

```diff
@@ -90,17 +90,17 @@ typedef struct {
 
 /* These are no longer used. */
 #if 0
-#define CO_GENERATOR_ALLOWED    0x1000
+#define CO_GENERATOR_ALLOWED    0x10000
```

Comment:
It sounds strange to keep an old macro **and** changes its value. I suggest to remove it.

Reply:
Agreed